src/include/common/constants.h

```diff
@@ -144,8 +144,9 @@ struct CopyConstants {
     static constexpr const char* ROW_IDX_COLUMN_NAME = "ROW_IDX";
     static constexpr uint64_t PANDAS_PARTITION_COUNT = 50 * DEFAULT_VECTOR_CAPACITY;
 
-    static constexpr const char* INT_CSV_PARSING_OPTIONS[] = {"SKIP"};
+    static constexpr const char* INT_CSV_PARSING_OPTIONS[] = {"SKIP", "SAMPLE_SIZE"};
     static constexpr uint64_t DEFAULT_CSV_SKIP_NUM = 0;
+    static constexpr int64_t DEFAULT_CSV_TYPE_DEDUCTION_SAMPLE_SIZE = -1;
```

Comment:
I think the default value should be 0 instead of -1

Reply:
Negative value means to scan the whole table for sniffing. This is inefficient, as suggested by previous experiences with the parameter, so I'll be setting it to 2048. Hopefully this doesn't break any tests.